cetmix_tower_server_queue/models/cx_tower_file.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class CxTowerFile(models.Model):
+    _inherit = "cx.tower.file"
+
+    is_processed = fields.Boolean(
+        string="Is Processed",
```

Comment:
From the [OCA Coding Guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#fields):
`
If the technical name of the field (the variable name) is the same to the string of the label, don't put string parameter for new API fields, because it's automatically taken. If your variable name contains "_" in the name, they are converted to spaces when creating the automatic string and each word is capitalized.
`